cpu/cc2538/Makefile.features

```diff
@@ -3,7 +3,7 @@ CPU_FAM = cc2538
 
 FEATURES_PROVIDED += periph_cpuid
 FEATURES_PROVIDED += periph_flashpage
-FEATURES_PROVIDED += periph_flashpage_raw
+FEATURES_PROVIDED += periph_flashpage_pagewise
```

Comment:
Is there a case where a CPU would *not* implement this feature?

Reply:
Yes, the stm32f2, f4 and f7. These have sectors with different flash ~~page~~ sector sizes (erases also per sector). The stm32f401re has 4 ~~pages~~ sectors of 16 KB, a 64 KB one and the rest is 128 KB large. Using flashpage_pagewise would be very complex as the amount of data provided varies with the sector number to be written, ignoring for a second that the pages are so big that it becomes impractical to have a full page in memory before writing it.